packages/accounts/plugindefs/types.ts

```diff
@@ -1,8 +1,9 @@
-import type { Abi, Address, Chain } from "viem";
+import type { ContractConfig } from "@wagmi/cli";
+import type { Abi, Chain } from "viem";
 
 export type PluginGenConfig = {
   abi: Abi;
-  address: Address;
+  address: ContractConfig["address"];
```

Comment:
doesn't this allow it to be either string or object now?

Reply:
thinking about this, we should be prescriptive about this and only allow Record<number, Address>

Reply:
also let's just call this field `addresses` it's a bit weird to see it called `address` and then a list of addresses